docs/ref/contrib/admin/index.txt

```diff
@@ -1362,6 +1376,14 @@ templates used by the :class:`ModelAdmin` views:
                 else:
                     return ['name']
 
+.. method:: ModelAdmin.get_orderable_by(request)
+
+    .. versionadded:: 1.10
+
+    The ``get_orderable_by`` method is given the HttpRequest and is expected to return a list or tuple of field names
```

Comment:
Missing backticks around HttpRequest. Wrap lines at 79 characters.
I think a common use case will be excluding one or more fields. For that case, it would be useful to document the pattern (untested):
```python
def get_orderable_by(self, request):
    return set(self.get_list_display(request)) - {'excluded_field'}
```